core/xchain/bin/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Contains build artifacts and other files that should not be committed.
+# Ignore all files and subdirectories in this directory.
+*
+!.gitignore
```

Comment:
@sergekh2 removed this from the root .gitignore yesterday https://github.com/river-build/river/pull/453/files
are you sure you need this?
maybe run ./scripts/yarn-clean.sh

Reply:
It's now put into core/run_files/bin, (delete file locally at this location)